app/views/admin/users/index/_controls.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= f.text_field :search, value: params[:search], class: "crayons-textfield mt-0", placeholder: "Search member...", aria: { label: "Search member by name, username, email, or Twitter/GitHub usernames" } %>
       </div>
       <div class="crayons-field flex-row items-center gap-2">
-        <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: true, prompt: "Filter by role..." }, class: "crayons-select mt-0" %>
+        <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: "All roles" }, class: "crayons-select mt-0" %>
```

Comment:
We've named the blank option in the dropdown as well as removed the prompt. You can read more about it [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1648452165993119)